exercises/resistor-color-trio/src/example.h

```diff
@@ -0,0 +1,36 @@
+#ifndef RESISTOR_COLOR_TRIO_H
+#define RESISTOR_COLOR_TRIO_H
+
+#include <stdint.h>
+
+typedef enum {
+   BLACK = 0,
+   BROWN = 1,
+   RED = 2,
+   ORANGE = 3,
+   YELLOW = 4,
+   GREEN = 5,
+   BLUE = 6,
+   VIOLET = 7,
+   GREY = 8,
+   WHITE = 9
+} resistor_band_t;
+
+typedef enum {
+   OHMS = 0,
+   KILOOHMS = 1,
```

Comment:
Amazingly I don't think I've ever seen this fully written out and it's really weird to see both "o"s back to back O_o

Reply:
I actually had to double check it it looked so weird!